content/monitors/faq/as-count-monitor-evaluation.md

```diff
@@ -0,0 +1,78 @@
+---
+title: as_count() monitor evaluation changes
+kind: faq
+---
+
+## Overview
+
+Currently all monitors that have `as_count` modifier are using a separate evaluation path that the other monitors, which can lead in certain use-cases to a different monitor behavior one could expect. **We intend to migrate monitor with `as_count` to the same evaluation path as all monitor. Here is the impact of such decisions.**  
```

Comment:
Currently all monitors that have the `as_count` modifier are using a separate evaluation path than other monitors. In certain use-cases this can result in unexpected behavior. 
**We intend to migrate all monitors with `as_count` to the same evaluation path as other monitors.**